### PR TITLE
build: update gradle and fix jitpack build failures

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
## Change list

update Gradle and fix Jitpack build failures
 
## Types of changes

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [ ] No changes in production code.
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -134,8 +130,6 @@ publishing {
             artifactId = 'java-client'
             version = '7.3.0'
             from components.java
-            artifact sourcesJar
```
Comment (mykola-mokhnach) [inline build.gradle:137]: why documentation and sources are excluded?

File: build.gradle [hunk 2]
```diff
@@ -60,33 +50,39 @@ compileJava {
 dependencies {
     compileOnly('org.projectlombok:lombok:1.18.12')
     annotationProcessor('org.projectlombok:lombok:1.18.12')
-    compile ("org.seleniumhq.selenium:selenium-java:${project.property('selenium.version')}") {
-        force = true
+    implementation ("org.seleniumhq.selenium:selenium-java") {
+        version {
+            strictly "${project.property('selenium.version')}"
```
Comment (mykola-mokhnach) [inline build.gradle:55]: 👍

**Comment by SrinivasanTarget** [comment]
https://jitpack.io/com/github/SrinivasanTarget/java-client/gradleUpgradeAndFixJitpack-4.1.2-ge7d2478-772/build.log

File: build.gradle (hunk 1 above)
Comment (SrinivasanTarget) [reply build.gradle:137]: components.java should include both sources and Javadoc as specified [here](https://github.com/appium/java-client/pull/1389/files#diff-c197962302397baf3a4cc36463dce5eaR33).

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**